ui/src/app/views/workspace/component.html

```diff
@@ -1,11 +1,27 @@
 <h2>AoU Research Portal Workspace</h2>
 <div *ngIf="user else loggedout">
-  <p class="description-text">Default workspace for July, 2017 demo</p>
+  <div *ngIf="!workspaceLoading; else workspaceLoadingDescription" class="description-text">
+    {{workspace.description}}
+  </div>
+  <ng-template #workspaceLoadingDescription>
+    <div class="description-text">
+      <span class="spinner spinner-inline"></span>
+      <span class="loading-text">Loading...</span>
+    </div>
+  </ng-template>
   <div class="area-block">
     <h3>CDR Version</h3>
     <div class="indented">
       <!-- TODO: Add CDR Version Info - blrubenstein -->
-      <p class="cdr-text">CDR version info goes here.</p>
+      <p *ngIf="!workspaceLoading; else workspaceLoadingCDR" class="cdr-text">
+        {{workspace.cdrVersionId}}
```

Comment:
This may be OK for now, but we aren't going to want to show the version ID (which is just a number) in the UI directly.

Instead we're going to want to grab a name from a list of CDR versions.

We can either adjust the API to return that (in addition to the ID) or have the client fetch a list of CDR versions with IDs and look it up. (We need the endpoint for fetching CDR versions regardless at some point.)